components/ui/icons/src/main/res/drawable/mozac_ic_device_desktop.xml

```diff
@@ -3,11 +3,12 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="16dp"
-    android:height="16dp"
-    android:viewportHeight="16"
-    android:viewportWidth="16">
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
```

Comment:
This icon has a different size. Is this intentional?

Reply:
Yes, I needed the size to match the other icons FFTV is using (which are all 24x24).

Reply:
Oh, I just noticed that this was 16dp **before**. Yep, makes sense!